orangecontrib/text/widgets/owwordcloud.py

```diff
@@ -121,7 +121,9 @@ def _new_webview(self):
 </html>'''
         if self.webview:
             self.mainArea.layout().removeWidget(self.webview)
-        self.webview = gui.WebviewWidget(self.mainArea, self, HTML, debug=True)
+        webview = self.webview = gui.WebviewWidget(self.mainArea, self, debug=False)
+        webview.setHtml(HTML)
+        self.mainArea.layout().addWidget(webview)
```

Comment:
Yes, functions that wrap gui components add them into the layout. But the QWidget-derived widgets should behave like the rest of Qt and _not_ add themselves.